src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -3782,6 +3784,7 @@ public object VisitSwitchStatement(SwitchStatementAst switchStatementAst)
                 //        if (sr != null) sr.Dispose();
                 //    }
 
+                bool isLiteralPath = (switchStatementAst.Flags & SwitchFlags.LiteralFile) == SwitchFlags.LiteralFile;
```

Comment:
Isn't there an [`Enum.HasFlag()`](https://docs.microsoft.com/en-us/dotnet/api/system.enum.hasflag?view=netframework-4.7.2) method for this?

Reply:
We can cache the method for now, but we'll need to undo it and save the bool as well when we switch to multi-file iteration in the next change (since the methods will have different return types) :)